src/tests/testSetup.ts

```diff
@@ -0,0 +1,60 @@
+import { runMutations } from "automutate";
+import fs from "node:fs";
+import path from "node:path";
+import ts from "typescript";
+
+import { fillOutRawOptions } from "../options/fillOutRawOptions.js";
+import { RawTypeStatOptions } from "../options/types.js";
+import { createTypeStatProvider } from "../runtime/createTypeStatProvider.js";
+
+export const runMutationForTest = async (
+	dirPath: string,
+	originalFileName = "original.ts",
+) => {
+	const typestatPath = path.join(dirPath, "typestat.json");
+	const projectPath = path.join(dirPath, "tsconfig.json");
+	const rawCompilerOptions = fs.readFileSync(typestatPath).toString();
```

Comment:
(e.g. for the `node:fs/promises` suggestion)

```suggestion
	const rawCompilerOptions = (await fs.readFile(typestatPath)).toString();
```